libp2p/NodeTable.cpp

```diff
@@ -548,6 +563,8 @@ void NodeTable::onPacketReceived(
                     if (it != m_allNodes.end())
                         it->second->lastPongSentTime = RLPXDatagramFace::secondsSinceEpoch();
                 }
+                else
```

Comment:
Looks a bit strange. Can you add a comment it is to skip `noteActiveNode()`?

Reply:
Maybe early return `if (!addNode(...)) return;` would look a bit nicer

Reply:
Good idea guys, will go with the early return since I agree that looks nicer / is more intuitive.